pkg/cmd/server/api/types.go

```diff
@@ -116,6 +116,11 @@ type NodeConfig struct {
 
 	// IPTablesSyncPeriod is how often iptable rules are refreshed
 	IPTablesSyncPeriod string
+
+	// ProjectEmptyDirQuota is the quota applied to emptyDir volumes specified in MB.
+	// This is currently implemented only for XFS, and is a limit per namespace per node.
+	// (this may change in the future)
+	ProjectEmptyDirQuota *int
```

Comment:
@deads2k apologies I'm not quite familiar with that, could you (or anyone else) fill me in on what that's a ref to?